src/App.js

```diff
@@ -1,19 +1,40 @@
 import React from 'react';
 import './App.css';
+import { NewTodo } from './NewTodo/NewTodo';
+import { TodoList } from './TodoList/TodoList';
 
 import users from './api/users';
+import todos from './api/todos';
```

Comment:
имена не должны пересекаться
```suggestion
import todosFromServer from './api/todos';
```